### PR TITLE
added CSS and getting the tab url

### PR DESCRIPTION
**Pull request by ajayVibinex**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,5 +1,55 @@
-const line = document.getElementsByClassName('blob-code blob-code-addition js-file-line');
 
-window.addEventListener('load', () => {
-    document.body.style.color = 'red';
-  });
\ No newline at end of file
+let url = window.location.toString(); 
+console.log('This is the url',url)
```
Comment (avikalpg) [inline main.js:3]: Remove unnecessary logs.

File: main.js [hunk 2]
```diff
@@ -1,5 +1,55 @@
...
+function ElementCreator(elementId,title,priority){
+  const newElement = document.createElement("span");
+  const  addTitle = document.createTextNode(title);
+  newElement.appendChild(addTitle);
+  newElement.setAttribute('id',priority);
+
+  const finalElement = document.getElementById(elementId);
+  finalElement.appendChild(newElement);
+}
+
```
Comment (avikalpg) [inline main.js:15]: This function is not used in your code anywhere. It can be removed.

File: main.js [hunk 3]
```diff
@@ -1,5 +1,55 @@
...
+
+function AddingCSS(elementId,title,priority){
+  one = document.head.appendChild(document.createElement("style"));
```
Comment (avikalpg) [inline main.js:18]: Please use a more descriptive variable name.

Comment (avikalpg) [reply main.js:18]: Please use a declaration, preferably `const`.

File: main.js [hunk 4]
```diff
@@ -1,5 +1,55 @@
...
+  {elementId:'issue_36407_link',title:'Low',priority:'Low'},
+  {elementId:'issue_36409_link',title:'Low',priority:'Low'},
+  {elementId:'issue_36410_link',title:'Low',priority:'Low'},
+
+  {elementId:'issue_16_link',title:'High',priority:'High'},
+  {elementId:'issue_10_link',title:'High',priority:'High'},
+  {elementId:'issue_9_link',title:'High',priority:'High'},
+  {elementId:'issue_2_link',title:'Medium',priority:'Medium'},
+  {elementId:'issue_8_link',title:'High',priority:'High'},
+]
```
Comment (avikalpg) [inline main.js:51]: Change the mock data to mimic the contract established with the backend.
Link: [Issue 47](https://github.com/Alokit-Innovations/devprofiler-cloudrun/issues/47)

**Review by avikalpg — CHANGES_REQUESTED** [review]

File: main.js [hunk 5]
```diff
@@ -1,5 +1,53 @@
...
+    content: 'Important';
+    color: white;
+    width: 12px;
+    height: 12px;
+    border: rgb(45, 0, 0);
+    border-radius: 5px;
+    margin-right: 10px;
+    padding-left: 5px;
+    padding-right: 5px;
+    padding-bottom: 2px;;}`;
```
Comment (ajayVibinex) [inline main.js:15]: remove the extra semicolon

File: main.js [hunk 6]
```diff
@@ -1,5 +1,53 @@
...
+};
```
Comment (ajayVibinex) [inline main.js:33]: Don't make two separate functions for a very similar task. Just take the "level" as parameter in the same function and modify the colors and content of the CSS accordingly.

File: main.js [hunk 7]
```diff
@@ -1,5 +1,53 @@
...
+  }
+];
+
+output[0].important.map((item)=>{
+  addingImportantCss(item)
+});
+
+output[0].relevant.map((item)=>{
+  addingRelevantCss(item)
+});
```
Comment (ajayVibinex) [inline main.js:53]: You don't need to write the same thing 2 times. Just make it a for loop over all the possible "level"s.
If needed, you can create an enum for levels.

File: main.js [hunk 8]
```diff
@@ -1,5 +1,53 @@
...
+    padding-right: 5px;
+    padding-bottom: 2px;;}`;
+};
+
+//sending url which contain repo name 
+// to do : getting user info for sending to backend
+let url = window.location.toString();
+
+// data getting from api 
+const output = [
```
Comment (ajayVibinex) [inline main.js:40]: use a better name than `output`. 
For example: `highlighted_PR_ids`

File: manifest.json [hunk 9]
```diff
@@ -7,11 +7,14 @@
...
     },
+    "permissions": [
+        "tabs"
+    ],
     "icons": {
         "128": "vibinex-logo.png"
     },
     "content_scripts":[{
-        "matches":["https://github.com/*/pull/*/files"],
+        "matches":["https://github.com/*pull*","https://github.com/*pulls*"],
```
Comment (ajayVibinex) [inline manifest.json:17]: check if  it work only for github.com/pulls or /pull

**Comment by avikalpg** [comment]
@ajayVibinex Please send me a message on Slack or Whatsapp when you fix the changes needed in this pull request.

File: main.js [hunk 10]
```diff
@@ -1,5 +1,37 @@
...
+function addingCssElement(elementId,status) {
+  let backgroundColor = status =='Important'?'rgb(61, 0, 0)':'rgb(86, 88, 0)';
+  let tagBackgroundColor = status =='Important'?'rgb(255,0,0)':'rgb(164, 167, 0)';
+  document.getElementById(`issue_${elementId}`).style.backgroundColor = backgroundColor;
+  let element = document.head.appendChild(document.createElement("style"));
+  element.innerHTML = `#issue_${elementId}_link::before{
+    background-color:${tagBackgroundColor};
+    content: '${status}';
+    background-color:${tagBackgroundColor};
+    content: '${status}';
```
Comment (tapishr) [inline main.js:10]: Delete this

**Review by tapishr — APPROVED** [review]

**Pull request merged**